src/stan/lang/generator/generate_log_prob.hpp

```diff
@@ -47,39 +48,41 @@ namespace stan {
 
       bool is_var_context = true;
       bool is_fun_return = false;
+      generate_try(2, o);
```

Comment:
Is this the new location of the try/catch that was removed from the `generate_located_statements` function?